use_case_examples/cifar/cifar_brevitas_training/README.md

```diff
@@ -85,7 +85,7 @@ Here, an image from the CIFAR10 data-set is randomly chosen and preprocessed. Th
 
 ## Hardware Used for the Experiment
 
-Experiments were conducted on an m6i.metal machine offering 128 CPU cores and 512GB of memory. The choice of hardware can significantly influence the execution time and performance of the model.
+Experiments were conducted on an m6i.metal machine offering 128 CPU cores and 512GB of memory. The choice of hardware can significantly influence the performance of the model.
```

Comment:
I would have kept "execution time" and remove performance no ? for me "performance" is always a bit ambiguous as it could also refer to its accuracy/metrics , which we are not talking about here !

Reply:
I agree.